src/index.tsx

```diff
@@ -1226,12 +1226,9 @@ export class App extends React.Component<any, AppState> {
                   }
                 };
 
-                const onPointerUp = (event: PointerEvent) => {
-                  this.setState({
-                    isResizing: false,
-                    resizingElement: null,
-                    selectionElement: null,
-                  });
+                const onPointerUp = () => {
+                  isDraggingScrollBar = false;
```

Comment:
these two lines belong on the last commit.

Reply:
Not sure what you mean.